package/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '3.4'
+
+x-defaults: &defaults
+  restart: always
+  image: raiden-services
+  env_file: services.env
+  volumes:
+    - /data/state:/state
+    - /data/keystore:/keystore
+
+services:
+  pfs:
+    << : *defaults
+    command: ["python3", "-m", "pathfinding_service.cli", "--enable-debug"]
+    labels:
+      - "traefik.enable=true"
+      - "traefik.frontend.rule=Host: pfs-ropsten-with-fee.services-dev.raiden.network"
```

Comment:
@ulope Can I somehow make this dynamic based on some variable in the `.env`  file?